packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -494,12 +495,17 @@ export default class Meetings extends WebexPlugin {
      * @memberof Meetings
      */
     async createMeeting(destination, type = null) {
+      // Validate that the provided destination is a personId and extract the UUID.
+      const targetDest = MeetingsUtil.determineIfDestinationIsPerson(destination) ?
```

Comment:
We already do this 

```
return MeetingInfoUtil.getSipUriFromHydraPersonId(hydraId.destination, webex).then((res) => {
      options.destination = res;
...
}
```

see `MeetingInfoUtil.generateOptions`

This changes the flow of the meetings plugin where the entire destination is determined at the meetingInfo level once it fetches and gets back the sipURI

Reply:
I investigated this method, and determined that a combination of `common#deconstructHydraId()` and `MeetingInfoUtils#getSipUriFromHydraPersonId()` works well in the updated implementation.